stratum/hal/lib/tdi/es2k/es2k_sde_meter.cc

```diff
@@ -0,0 +1,248 @@
+// Copyright 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+// ES2K-specific PktModMeterConfig methods.
```

Comment:
```suggestion
// ES2K-specific PktModMeter methods.
```
The methods work on PktMeters. PktModMeterConfig is a supporting player.

Reply:
Fixed